codegen/headerparser.py

```diff
@@ -345,7 +348,7 @@ def __init__(self, argAsString, cinput=True):
 
 if __name__ == '__main__':
     THISDIR = os.path.abspath(os.path.dirname(__file__))
-
+    print(THISDIR)
```

Comment:
Extra print @kmuehlbauer ?

Reply:
@djhoese Yes, that's where I started to look, I'll remove it...